webhook/wireupwebhook.go

```diff
@@ -71,9 +71,9 @@ func WireUpWebhook(clt client.Client, mgr manager.Manager, whk *webhook.Server,
 	return GenerateWebhookCerts(clt, certDir)
 }
 
-//assuming we have a service set up for the webhook, and the service is linking
-//to a secret which has the CA
-func WireUpWebhookSupplymentryResource(ctx context.Context, mgr manager.Manager, wbhSvcName, validatorName, certDir string, caCert []byte) {
```

Comment:
It seems the certDir is never used inside of the function :-)  Since there are some app webhook changes, could you verify if a new ACM application (kind: ) can be applied without error? 
```
apiVersion: app.k8s.io/v1beta1
kind: Application
```

The app validation webhook does some checks when a app is created.